src/Controller/TodoController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Controller;
+
+use App\DataFixtures\AppFixtures;
+use App\Repository\TodoRepository;
+use Doctrine\Persistence\ObjectManager;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class TodoController extends AbstractController
+{
+    private TodoRepository $todoRepository;
+
+    public function __construct(
+        TodoRepository $todoRepository,
+    ){
+        $this->todoRepository = $todoRepository;
+    }
+
+    /**
+     * @Route("/todo", name="todo", methods={"GET"})
+     */
+    public function index(): Response
+    {
+        $todos = $this->todoRepository->findAll();
```

Comment:
Take sorted todos by created at DESC

Reply:
https://stackoverflow.com/questions/17116888/how-to-sort-findall-doctrines-method